plugins/node/instrumentation-tedious/src/instrumentation.ts

```diff
@@ -78,7 +78,7 @@ export class TediousInstrumentation extends InstrumentationBase {
     return [
       new InstrumentationNodeModuleDefinition(
         TediousInstrumentation.COMPONENT,
-        ['>=1.11.0 <18'],
+        ['>=1.11.0 <19'],
```

Comment:
please also update the supported version range in the README of the instrumentation.

Reply:
Will do